include/iceflow/consumer-tlv.hpp

```diff
@@ -271,35 +272,35 @@ class ConsumerTlv {
         std::vector<std::vector<ndn::Block>> splitManifestBlocks;
 
         // grouping manifest data according to type
-        for (int i = 0; i < m_manifestDataTypes.size(); i++) {
+        for (int m_manifestDataType : m_manifestDataTypes) {
```

Comment:
`m_manifestDataType` is only used locally, right? Then I would remove the `m_` prefix as that would indicate that it is a class member variable.